database/v2/seeders/20191021130637-demo-user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const hashHelpers = require('../../../api/helpers/hashHelpers');
+
+async function createMockUser(i) {
+    const data = await hashHelpers.createHash("password");
+    const obj = {
+      userName: `John Doe`,
+      email: `demo${i}@demo.com`,
+      password: data,
+      role: 'User',
+      locale: 'ru',
+      resetPasswordToken: null,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    };
+    return obj;
+}
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    
+    const dataTemp = await Promise.all(Array(10).fill(1).map((el, i) => {
+      return createMockUser(i).then(obj => obj);
```

Comment:
Use meaningful variables in array methods: el => user, i => index.
Replace obj to smth else.

Reply:
Done